Content.Server/Destructible/Thresholds/Behaviors/SpawnEntitiesBehavior.cs

```diff
@@ -20,7 +20,7 @@ public sealed class SpawnEntitiesBehavior : IThresholdBehavior
 
         public void Execute(EntityUid owner, DestructibleSystem system)
         {
-            var position = system.EntityManager.GetComponent<TransformComponent>(owner).MapPosition;
+            var position = system.EntityManager.GetComponent<TransformComponent>(owner).Coordinates;
```

Comment:
Pretty sure this will have issues with parenting, e.g. if you're in a container it will attach to the parent.